crates/vault-registry/src/lib.rs

```diff
@@ -459,43 +486,74 @@ impl<T: Config> Module<T> {
         Ok(btc_address)
     }
 
-    /// checks if there are sufficient issued tokens that can be replaced. If so, it
-    /// is increased.
     pub fn try_increase_to_be_replaced_tokens(
         vault_id: &T::AccountId,
         tokens: PolkaBTC<T>,
-    ) -> Result<(), DispatchError> {
+        griefing_collateral: DOT<T>,
+    ) -> Result<(PolkaBTC<T>, DOT<T>), DispatchError> {
         ext::security::ensure_parachain_status_running::<T>()?;
         let mut vault = Self::get_active_rich_vault_from_id(&vault_id)?;
 
-        let new_to_be_replaced = vault
+        let new_tokens = vault
```

Comment:
leaving is at `new_to_be_replaced` makes it a bit clearly IMO where those tokens will be added